.github/workflows/maven-run-tests.yml

```diff
@@ -38,3 +38,9 @@ jobs:
         GITHUB_TOKEN: ${{ github.token }}
     - name: Build Examples with Maven
       run: mvn -B package --file main/basyx.examples/pom.xml
+    - name: Build AasWrapper with Maven
+      run: mvn -B package --file main/basyx.aasWrapper/pom.xml -Dmaven.javadoc.skip=true
```

Comment:
Why is javadoc skipped here? Are there any issues with it?

Reply:
Dear Frank,

Yes the build was giving errors as follows:

" Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:javadoc (generate-javadoc) on project basyx.components.AASWrapper: An error has occurred in Javadoc report generation: 
Error:  Exit code: 1 - /home/runner/work/basyx-java-examples/basyx-java-examples/main/basyx.aasWrapper/src/main/java/org/eclipse/basyx/wrapper/provider/GenericHTTPInterface.java:58: warning: no @param for backend
Error:  	public GenericHTTPInterface(IWrapperProvider backend) { "

The possible cause is the javadoc version i.e 3.2.0.

Following is the link :
https://stackoverflow.com/questions/23542876/unable-to-build-maven-project-due-to-javadoc-error

The other option would be to update the javadoc version.

Reply:
Understood, this will be fixed in a later step. Many thanks for pointing this out.